src/zkevm_specs/evm/instruction.py

```diff
@@ -123,7 +124,10 @@ def constrain_step_state_transition(self, **kwargs: Transition):
         ), f"Invalid keys {list(set(kwargs.keys()).difference(keys))} for step state transition"
 
         for key, transition in kwargs.items():
+            # for key in keys:
             curr, next = getattr(self.curr, key), getattr(self.next, key)
+            # transition = kwargs.get(key, Transition.same())
+            # print(key, curr, next)
```

Comment:
nitpick: Accidental comment?
```suggestion
            curr, next = getattr(self.curr, key), getattr(self.next, key)
```

Reply:
Yes, I was testing if we should check all keys instead of the keys only in the `kwargs`. But after discussing with @han0110, he thinks it'll be unnecessary to check all keys in the step state.